docs/man/rpmkeys.8.md

```diff
@@ -41,29 +41,25 @@ armored public key can be added to the **rpm** database using
 ring management is performed exactly like package management. For
 example, all currently imported public keys can be displayed by:
 
-**rpm -q gpg-pubkey**
-
-A more convenient way to display them is
-
 **rpmkeys** **\--list**
 
-More details about a specific public key, when imported, can be displayed by
-querying. Here\'s information about the Red Hat GPG/DSA key:
-
-**rpm -qi gpg-pubkey-db42a60e**
+This lists all currently imported public keys (aka certificates)
```

Comment:
A small suggestion: I try to to not start paragraphs with a demonstrative pronoun (this or that) and instead name the thing that I'm talking about.  My experience is that this makes scanning easier as the reader doesn't have to decipher the referent.  So in this case, I would suggest "This command."

Reply:
Actually the rest of the manual has imperatives, so following that this would be just 'List all currently...', but actually that's wrong too, because it optionally takes arguments. Argh :laughing: